lib/task/tools/settingsTask.class.php

```diff
@@ -0,0 +1,351 @@
+<?php
+
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * Set or get setting values.
+ *
+ * @author  Mike Cantelon <mike@artefactual.com>
+ */
+class settingsTask extends arBaseTask
+{
+    protected $ormClasses;
+
+    public function __construct(sfEventDispatcher $dispatcher, sfFormatter $formatter)
+    {
+        $this->setOrmClasses([
+            'setting' => QubitSetting::class,
+        ]);
+
+        parent::__construct($dispatcher, $formatter);
+    }
+
+    public function setOrmClasses(array $classes): void
+    {
+        $this->ormClasses = $classes;
+    }
+
+    /**
+     * @see sfTask
+     *
+     * @param mixed $arguments
+     * @param mixed $options
+     */
+    public function execute($arguments = [], $options = [])
+    {
+        parent::execute($arguments, $options);
+
+        $this->validateOptions($arguments, $options);
+
+        $this->displayCurrentSettings();
```

Comment:
When I run the task I always get the full list of settings, e.g.

```
[dev/issue-13604-settings-task][~/atom]$ symfony tools:setting get version
>> sfPluginAdminPluginConfiguration The plugin "arDominionB5Plugin" does not exist.
------------------------------------------------------------------------------------
Name                                                              Scope
------------------------------------------------------------------------------------
_csrf_token                                                       element_visibility
accession_counter
accession_mask
accession_mask_enabled
actor                                                             default_template
...
```

Is this intentional? It makes it hard to actually see the value for the selected setting.

Reply:
The "_csrf_token" is a bug (https://projects.artefactual.com/issues/13613) I just discovered while testing this task. 😆

Reply:
Oh, good catch @djjuhasz!

Reply:
Weird one, that "_csrf_token". 😨 

Yeah, I added that display of a list of settings to make the settings easier to work with, but displaying it the way I'm displaying it - with every command - is kludge-y. I'll take it out for now... I could make a new CLI task to display settings if need be. That'd be less weird.